src/plugins/dashboard/public/application/_dashboard_app.scss

```diff
@@ -61,16 +61,3 @@
     flex-direction: column;
   }
 }
-
-// Temporary fix for two tone icons to make them monochrome
-.dshSolutionToolbar__editorContextMenu--dark {
-  .euiIcon path {
-    fill: $euiColorGhost;
-  }
-}
-
-.dshSolutionToolbar__editorContextMenu--light {
-  .euiIcon path {
-    fill: $euiColorInk;
-  }
```

Comment:
I removed the logic from `top_nav/editor_menu.tsx` that relied on these special classes because the workaround no longer seems to be necessary - regardless of the theme, the icons in the editor menu are all monochrome as can be seen here: 

![image](https://user-images.githubusercontent.com/8698078/189212788-3fbd0818-e955-4d93-b1b0-919027871a4c.png)

Therefore, these classes are no longer necessary 👍